pkg/kubelet/pod_workers.go

```diff
@@ -291,9 +291,9 @@ func killPodNow(podWorkers PodWorkers, recorder record.EventRecorder) eviction.K
 		}
 
 		// we timeout and return an error if we don't get a callback within a reasonable time.
-		// the default timeout is relative to the grace period (we settle on 2s to wait for kubelet->runtime traffic to complete in sigkill)
+		// the default timeout is relative to the grace period (we settle on 3s to wait for kubelet->runtime traffic to complete in sigkill)
```

Comment:
According to [derek's comment on the issue](https://github.com/kubernetes/kubernetes/issues/41347#issuecomment-282175790), 2s was an initial estimate.  It is the time for the runtime kill to return.  It essentially boils down to a call to kubelet.go:syncPod()